flaky-tests/generating-junit-reports.md

```diff
@@ -21,3 +21,87 @@ $ npm install jest-junit --save-dev
 ```
 
 With this configuration, Jest runs will by default output a `junit.xml` file in the working directory. To further configure the reporter, consult the detailed [documentation on GitHub](https://github.com/jest-community/jest-junit?tab=readme-ov-file#jest-junit).
+
+## Mocha
```

Comment:
Similar for these other runners, we should make sure the document specifies how to populate the `file` field in the xml. For `jest` it is an environment variable, for `gtest` it depends on version, and IIRC for pytest you need to use `junit_family = xunit2` in the configuration.